ERMize/topics/Entity/Naming-Identifiers.topic

```diff
@@ -5,14 +5,27 @@
        xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
        title="Naming Identifiers" id="Naming-Identifiers">
 
-    <available-only-for instance="ermize">Standard</available-only-for>
-
+    <available-only-for>Standard</available-only-for>
+    <p>Identifiers are commonly used in conceptual data models as they provide more clarity, helping stakeholders to understand what each attribute or entity represents.
+        By using identifiers, the user can enhance the documentation of the data model.
+    </p>
     <chapter title="Code" id="code">
+        <p>
+            We have added the functionality to add identifiers to entities.
+            See the code snippet below how this is done in code.
+        </p>
+        <note>The user has the ability to add identifiers when no attributes exist in the entity.</note>
         <code-block src="naming-identifiers.txt" />
+
     </chapter>
 
     <chapter title="Output" id="output">
-        <note>This is a concept image and not the actual result of the application.</note>
+        <p>This is output image that's being generated by ERMize based on the code snippet above.</p>
+        <note>
+            <p>This is a concept image and not the actual result of the application.</p>
```

Comment:
Ik weet dat ik dit als suggestie heb gedaan, maar wil er toch iets anders mee doen 😓

Dit ziet er voor de gebruiker namelijk raar uit aangezien er eerst gezegd wordt dat dit het resultaat is dat gegenereerd wordt, maar daarna staat er dat dit een concept is en niet een echt resultaat. Heb je ideeën hiervoor?

Reply:
Hier hoef je niet meer op te letten. Dit gaat vanzelf opgelost worden als de code voor deze functionaliteit toegevoegd is aan de API

Reply:
Dan hoeft die note dan toch niet?

Reply:
dat klopt! haal maar weg!